src/contrib/mlir/backend/cpu/cpu_backend.cpp

```diff
@@ -188,25 +188,42 @@ void MLIRCPUBackend::lowerNgDialect()
 
     NGRAPH_CHECK(m_module, "MLIR module is not ready.");
 
-    // Lower Standard dialect to LLVM dialect.
-    mlir::LLVMTypeConverter llvmConverter(&m_context);
-    mlir::OwningRewritePatternList patterns;
-    mlir::populateLoopToStdConversionPatterns(patterns, &m_context);
-    mlir::populateStdToLLVMConversionPatterns(llvmConverter, patterns);
-
-    mlir::ConversionTarget target(m_context);
-    target.addLegalDialect<mlir::LLVM::LLVMDialect>();
-    target.addLegalOp<mlir::ModuleOp, mlir::ModuleTerminatorOp>();
-    target.addDynamicallyLegalOp<mlir::FuncOp>(
-        [&](mlir::FuncOp op) { return llvmConverter.isSignatureLegal(op.getType()); });
-
-    auto result =
-        mlir::applyFullConversion(m_module.get(), target, std::move(patterns), &llvmConverter);
-    NGRAPH_CHECK(succeeded(result), "Standard to LLVM dialect conversion failed");
+    lowerStandardDialect();
+}
+
+// Lower Standard dialect to LLVM dialect
+void MLIRCPUBackend::lowerStandardDialect()
+{
+    // lamda call backs to collect a set of patterns to convert from the Standard dialect to LLVM.
+    auto patternListFiller = [&](mlir::LLVMTypeConverter& converter,
+                                 mlir::OwningRewritePatternList& patterns) -> void {
+        mlir::populateStdToLLVMConversionPatterns(converter, patterns);
+    };
+
+    // lamda callbacks to create an instance of LLVMTypeConverter in the given context.
+    auto typeConverterMaker =
+        [&](mlir::MLIRContext* context) -> std::unique_ptr<mlir::LLVMTypeConverter> {
+        return std::make_unique<mlir::LLVMTypeConverter>(context);
+    };
+
+    mlir::PassManager pm(&m_context);
+    pm.addPass(mlir::createLowerToLLVMPass(patternListFiller, typeConverterMaker));
```

Comment:
Did you try using `createLowerToLLVMPass` without the typeConverterMaker? The default type converter should work. I wonder why there is no version with a default pattern list. Maybe we could contribute that!

Reply:
Oh, wait! That version already exists. I missed it:
```
/// Creates a pass to convert the Standard dialect into the LLVMIR dialect.
std::unique_ptr<OpPassBase<ModuleOp>> createLowerToLLVMPass();
```